Automattic/wp-calypso
# BlazePress - Hotlink campaings via URL

Related to #

WARNING: This PR should be merged after  https://github.com/Automattic/wp-calypso/pull/74342

## Proposed Changes

- Create a campaign url hotlink system. The url format is similar to this one:
http://calypso.localhost:3000/advertising/javiolmo89.wordpress.com/campaigns?campaign_id=10 
You need to pass a desired campaign_id. If it exists it will automatically jump the page to that url. But only once on campaign load.

## Testing Instructions
- Create multiple campaigns (so you can see at least some scroll in the campaign list in advertising section
- Create a hotlink url with the campaign id similar as: http://calypso.localhost:3000/advertising/javiolmo89.wordpress.com/campaigns?campaign_id=10 
- the browser should OPEN and scroll the page to the position where this campaign is. It should also work on mobile version

![hotlink](https://user-images.githubusercontent.com/43957544/224709625-c664c0e1-9fce-4b77-8f54-751095feb1ff.gif)

## Pre-merge Checklist
